.devcontainer/install-dependencies.sh

```diff
@@ -24,6 +24,17 @@ set -eu
 # -s --skip-installed   : Skip anything that's already installed
 #
 
+#
+# This file includes documentation in lines prefixed `#doc#`.
+# These lines are extracted by running `task doc:dependencies` from the root folder.
+#
+# Each depencency should be identified by a single line with the following format:
+#
+# | <name> | <version> | <details> |
```

Comment:
Missing the `#doc` prefix?

Reply:
No, this is documentation on how to use it - the table header is in another file.